sites/skeleton.dev/src/examples/components/toast/Example.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+
+import { Toaster, createToaster } from '@skeletonlabs/skeleton-react';
+
+export default function () {
+	const toaster = createToaster();
+	return (
+		<>
+			<Toaster toaster={toaster}></Toaster>
+			<button
+				className="btn preset-filled"
+				onClick={() => {
+					toaster.info({
+						title: 'This is a toast!'
+					});
+				}}
+			>
+				Toast!
```

Comment:
Go ahead and nix the exclamation marks or the trigger buttons. We don't do this in other examples.